app/models/work.rb

```diff
@@ -0,0 +1,30 @@
+class Work < ApplicationRecord
+  has_many :votes, dependent: :destroy
+
+  # validations goes here: user and work combo is unique...
+  validates :title, presence: true, uniqueness: {scope: :creator, :message => "and creator combination already exists"}
+  validates :creator, presence: true
+  validates :publication_year, presence: true, numericality: {only_integer: true, greater_than: 0, less_than_or_equal_to: Date.today.year + 1}
```

Comment:
Fancy! ...Were there requirements around this? Regardless, it's well tested, so I'm not complaining :)